Texstyle/Texstyle/Text.swift

```diff
@@ -16,7 +16,7 @@ public final class Text {
     }
 
     /// Clears cached attributed string.
-    private var substyles: [TextSubstyle] = [] {
+    var substyles: [TextSubstyle] = [] {
```

Comment:
Why did you make this variable not `private`?

Reply:
To use it in `Text+Operators.swift`. Actually, it has `internal` visibility and will not be available outside.